### PR TITLE
skype => 8.69.76.76

### PR DESCRIPTION
**Pull request by supechicken666**
depends_on #5301

Tested on `x86_64`

### CODE REVIEW

File: packages/skype.rb [hunk 1]
```diff
@@ -3,36 +3,26 @@
...
-    depends_on 'alien' => :build
+  
+  if ARCH == 'x86_64' then
     depends_on 'gtk3'
     depends_on 'sommelier'
   end
 
   binary_url ({
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/skype-8.68.76.35-chromeos-x86_64.tar.xz',
+     x86_64: "https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_#{version}_amd64.deb"
```
Comment (uberhacker) [inline packages/skype.rb:15]: source_url, not binary_url?

File: packages/skype.rb [hunk 2]
```diff
@@ -3,36 +3,26 @@
...
-
-  def self.build
-    system 'curl https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_8.68.76.35_amd64.deb -#o skype.deb'
-    puts "Archive downloaded".lightgreen
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read('skype.deb') ) == '701449b86ff1c10ee800cc017033c7c2e9b31023fcc1a9058590babf070ce3bb'
-    system 'alien -tc skype.deb'
-    system "tar xvf skypeforlinux-#{version}.tgz"
-  end
+    
+  is_external
```
Comment (uberhacker) [inline packages/skype.rb:21]: Not sure about this.  What advantage does it provide?

File: packages/skype.rb [hunk 3]
```diff
@@ -3,36 +3,26 @@
...
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}"
-    FileUtils.mv 'usr/bin', "#{CREW_DEST_PREFIX}"
-    FileUtils.mv 'usr/share', "#{CREW_DEST_PREFIX}"
-    FileUtils.ln_s "#{CREW_PREFIX}/bin/skypeforlinux", "#{CREW_DEST_PREFIX}/bin/skype"
+    FileUtils.mkdir_p CREW_DEST_PREFIX
+    FileUtils.ln_s "#{CREW_PREFIX}/bin/skypeforlinux", 'usr/bin/skype'
```
Comment (uberhacker) [inline packages/skype.rb:25]: Probably should mv before ln_s for clarity.

Comment (supechicken666) [reply packages/skype.rb]: Moved `ln` to `preinstall` part

File: packages/skype.rb (hunk 2 above)
Comment (supechicken666) [reply packages/skype.rb:21]: It provides an easier way to add different arch of deb (if arch supported) :)

**Comment by supechicken** [comment]
> Can we remove this and see how it works?

Removed

**Pull request merged**